docs/src/cli/install.md

```diff
@@ -253,7 +253,7 @@ your MacOS or Linux machine.
 
 - Follow instructions at: https://formulae.brew.sh/formula/solana
 
-[Homebrew formulae](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/solana.rb)
+[Homebrew formulae](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solana.rb)
```

Comment:
Hmm, while the current link is invalid, the version listed in the new file is ancient AND there is a deprecation warning:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solana.rb#L20C3-L20C57

The deprecation warning is valid given that we are using this repo and not:
https://github.com/solana-labs/solana

I'm not sure if there is an entry for this repo, but that seems like it'd be the more proper fix

Reply:
Remove for now, I'll figure out an alternative solution